lib/elixir_auth_google.ex

```diff
@@ -145,7 +188,7 @@ defmodule ElixirAuthGoogle do
     # https://stackoverflow.com/questions/31990134
   end
 
-  defp google_client_id do
+  def google_client_id do
```

Comment:
I think I would have keep this function private. The users already know this id as they have defined in as an environment variable. I understand you added it public to be able to run the tests but I would have change sightly the tests to not use this function.

Reply:
The reason I'm not concerned with "exposing" the `google_client_id` is that it's right there in the URL when a request is made to the Google API. i.e. it's _public_ info. wouldn't _dare_ do this for the `google_client_secret`. 💭